src/Remote/Composer.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+namespace Castor\Remote;
+
+use Castor\Console\Application;
+use Castor\GlobalHelper;
+use Castor\PathHelper;
+use Castor\Remote\Exception\ComposerError;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Console\Helper\ProgressIndicator;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Process\ExecutableFinder;
+use Symfony\Component\Process\Process;
+
+use function Castor\fingerprint;
+
+class Composer
+{
+    public const VENDOR_DIR = '/.castor/vendor/';
+    public const DEFAULT_COMPOSER_CONFIGURATION = [
+        'description' => 'This file is managed by Castor. Do not edit it manually.',
+        'config' => [
+            'sort-packages' => true,
+        ],
+        'replace' => [
+            'castor/castor' => Application::VERSION,
```

Comment:
🤯 ! smart!

Reply:
An idea from Joel :upside_down_face: 

And one additional benefit is that remote packages will be updated as soon users update their Castor version (because the composer.json will change) :tipping_hand_man: